views/security/login.php

```diff
@@ -33,37 +35,46 @@
             </div>
             <div class="panel-body">
                 <?php $form = ActiveForm::begin([
-                    'id'                     => 'login-form',
-                    'enableAjaxValidation'   => true,
+                    'id' => 'login-form',
+                    'enableAjaxValidation' => true,
                     'enableClientValidation' => false,
-                    'validateOnBlur'         => false,
-                    'validateOnType'         => false,
-                    'validateOnChange'       => false,
+                    'validateOnBlur' => false,
+                    'validateOnType' => false,
+                    'validateOnChange' => false,
                 ]) ?>
 
-                <?= $form->field(
-                    $model,
-                    'login',
-                    ['inputOptions' => ['autofocus' => 'autofocus', 'class' => 'form-control', 'tabindex' => '1']]
-                ) ?>
+                <?php
+                if ($module->debug) {
+                    echo $form->field($model, 'login', ['inputOptions' => ['autofocus' => 'autofocus', 'class' => 'form-control', 'tabindex' => '1']])->dropDownList(
+                        ArrayHelper::map(User::find()->where(['blocked_at' => null])->all(), 'username', 'username'));
```

Comment:
I also don't like quering results from database inside view files. Maybe move this to the model method, and use getter here (e.g. `$model->getLoginList()`).